src/main/scala/com/example/reporters/common/SimpleLogReporter.scala

```diff
@@ -0,0 +1,18 @@
+package com.example.reporters.common
+
+import java.io
+import java.io.File
+
+trait SimpleLogReporter extends Reporter {
+  val reportData: ReportUnit
+
+  override def report(): Unit = {
+    val filename = System.getenv("REPORT_LOCATION") + reportData.name
+    new File(filename).delete
+    new io.PrintWriter(filename) {
```

Comment:
not the best way to write data. does it support s3? please consider rewriting this. If you need help, please let me know